simpleTest2/appSecondPb.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function (array) {
+
+  	var firstMAX = Number.NEGATIVE_INFINITY;
+    var secondMAX = Number.NEGATIVE_INFINITY;
+
+    if(array.length === 0) return 0;
+
+    for(var index = 0; index < array.length; index++) {
+            if(array.length === 1) return [array[index],0];
```

Comment:
This can be pulled up.

Reply:
Now I see that I do not need this as I am already printing [1,-Infinity] if I have just one item like [1].